main.py

```diff
@@ -1,27 +1,101 @@
 import numpy as np
 import matplotlib.pyplot as plt
+from nltk.corpus import brown
+from linalgo.client import LinalgoClient
 
+token = "e5a30d4fa8b0b52a0513363c5582a214deb255b4"
```

Comment:
Usually, it's dangerous to commit a valid authentication token on GitHub and is considered a security hazard. You could use an ENV variable here.